docs/sansio.rst

```diff
@@ -242,7 +242,7 @@ that are provided to you. Continuing from the example in the Requests_ section::
 Utilities
 ---------
 
-.. function:: format_url(url, url_vars)
+.. function:: format_url(url, url_vars, base_url=None)
```

Comment:
Is it worth making this keyword-only? E.g. is `format_url("/some/url/path", {}, "api.github.com/somewhere")` clear enough what all the arguments represent? I'm leaning towards keyword-only, but I would like to hear your opinion first.

Reply:
And should the default just be `"https://api.github.com/"`? What does supporting `None` by us?

Reply:
ok yeah making it keyword only works. I will make the change.